src/SCREENS.ts

```diff
@@ -242,6 +242,7 @@ const SCREENS = {
             QUICKBOOKS_ONLINE_INVOICE_ACCOUNT_SELECTOR: 'Policy_Accounting_Quickbooks_Online_Invoice_Account_Selector',
             XERO_IMPORT: 'Policy_Accounting_Xero_Import',
             XERO_ORGANIZATION: 'Policy_Accounting_Xero_Customers',
+            XERO_CHART_OF_ACCOUNTS: 'Policy_Accounting_Xero_Import_Chart_Of_Accounts',
```

Comment:
Was this `Import` added intentionally? 
```suggestion
            XERO_CHART_OF_ACCOUNTS: 'Policy_Accounting_Xero_Chart_Of_Accounts',
```

Reply:
Yes, it's intended. Quickbooks Chart of accounts page has the same naming and I'm trying to keep the same convention 😄